content/en/logs/processing/attributes_naming_convention.md

```diff
@@ -1,143 +1,147 @@
 
-To enforce standard attributes, administrators have the right to re-copy an existing set of non-standard attributes into a set of standard ones. This enables noncompliant logs sources to become compliant without losing any previous information.
+Aliasing is most particularly useful when it comes to filtering or aggregating logs flowing from different sources altogether - that is to say when [turned into facets][1]. Gathering content from multiple and heterogenous sources into a unique **Standard Facet** makes it much more straightforward to build insights or pivot information across your organisation.
 
-### Standard attributes in Log Explorer
+As a matter of example, follow the clients most impacted by latencies on a hybrid [Apache][2] and [Amazon Cloud Front][3] infrastructure, using the Standard `Network Client IP` facet alonside the Standard `duration`.
 
-Typically, during a transitional period, standard attributes may coexist in your organization along with their non-standard versions. To help your users cherry-pick the standard attributes in this context, they are identified as such in the explorer (e.g. in the facet list, or in measure or group selectors in Analytics).
+### Curate Standard Attributes
 
-{{< img src="logs/processing/attribute_naming_convention/standard_attribute_explorer.png" alt="Standard Attributes"  style="width:60%;">}}
+Log Integrations natively rely on a [default set](#default-standard-attribute-list) of standard attributes.
 
-If you are an administrator or prescriptor of the naming convention in your organization, you can take this opportunity to educate other users with standard attributes, and nudge them to align.
+But Admin users in your organisation are entitled to curate the list:
 
-## Standard attribute list
+- either from the [Log Explorer][1], **promoting** existing attributes as Standard Attributes.
+- or from the Standard Attribute [Configuration Page](#standard-attributes-in-explorer) **creating** new Standard Attributes from scratch.
 
-The standard attribute table comes with a set of [predefined standard attributes](#default-standard-attribute-list). You can append that list with your own attributes, and edit or delete existing standard attributes:
+### Aliasing
 
-{{< img src="logs/processing/attribute_naming_convention/edit_standard_attributes.png" alt="Edit standard attributes"  style="width:80%;">}}
+Aliasing a source attribute towards a destination attribute allow logs carrying the source attribute to carry the source and destination attribute, both with the same value.
 
-### Add or update standard attributes
+Users can interact with either the aliased (source) or standard (destination) faceted attribute. But as far as facets are concerned, users are [nudged][4] to use the Standard Facet rather than the Aliased one. This provides guidance towards the naming convention, and smoothly discourages from building assets (such as saved views or dashboards) based on non-standard content.
```

Comment:
```suggestion
Users can interact with either the aliased (source) or standard (destination) faceted attribute. As far as facets are concerned, however, users are [nudged][4] to use the standard facet rather than the aliased one. This provides guidance towards the naming convention, and discourages users from building assets (such as saved views or dashboards) based on non-standard content.
```